server/routes/api/__test__/auth.unit.test.js

```diff
@@ -0,0 +1,56 @@
+const request = require('supertest');
+const app = require('../../../../app');
+const User = require('../../../models/user');
+
+describe('POST Auth/Login', () => {
+  const emailForTesting = 'testUser@test.com';
```

Comment:
Let's be more specific with this variable name. Maybe something like `existingUserEmail` that identifies that this email address is attached to a user that's already created.